Makefile

```diff
@@ -237,3 +238,47 @@ catalog-push: ## Push a catalog image.
 # Run CI steps
 .PHONY: ci
 ci: test
+
+.PHONY: api-docs
+api-docs: crdoc kustomize
+	@{ \
+	set -e ;\
+	TMP_DIR=$$(mktemp -d) ; \
+	$(KUSTOMIZE) build config/crd -o $$TMP_DIR/crd-output.yaml ;\
+	$(CRDOC) --resources $$TMP_DIR/crd-output.yaml --output docs/api.md ;\
+	}
+
+
+CRDOC = $(shell pwd)/bin/crdoc
+.PHONY: crdoc
+crdoc: ## Download crdoc locally if necessary.
+	$(call go-get-tool,$(CRDOC), fybrik.io/crdoc,$(CRDOC_VERSION))
+
+
+.PHONY: kustomize
+kustomize: ## Download kustomize locally if necessary.
```

Comment:
Could you add the logic to not download the binary if not needed?
[Reference](https://v1-23-x.sdk.operatorframework.io/docs/upgrading-sdk-version/v1.23.0/#gov3-fix-makefile-targets-to-not-download-binaries-when-they-are-found-already)

Reply:
done

Reply:
It would be great if for tools we could be great if we could check if the right version of a tool is installed and if not then go get it.  Maybe the bingo too that @iblancasa mentioned above could be used for this.